src/app/modules/companies/constants/pageInfo.ts

```diff
@@ -0,0 +1,7 @@
+export const pageInfo = {
```

Comment:
Maybe better move this consts to a common level.

Reply:
Currently we have separate settings for each module. We have a user story in backlog to control such things on a higher level